x/ccv/consumer/keeper/validators.go

```diff
@@ -102,11 +104,20 @@ func (k Keeper) ValidatorByConsAddr(sdk.Context, sdk.ConsAddress) stakingtypes.V
 
 // Slash queues a slashing request for the the provider chain
 // All queued slashing requests will be cleared in EndBlock
-func (k Keeper) Slash(ctx sdk.Context, addr sdk.ConsAddress, infractionHeight, power int64, _ sdk.Dec, infraction stakingtypes.InfractionType) {
+func (k Keeper) Slash(ctx sdk.Context, addr sdk.ConsAddress, infractionHeight, power int64, slashFactor sdk.Dec, infraction stakingtypes.InfractionType) {
 	if infraction == stakingtypes.InfractionEmpty {
 		return
 	}
 
+	// If this is a previously standalone chain and infraction happened before the changeover was completed,
+	// slash only on the standalone staking keeper.
+	if k.IsPrevStandaloneChain() && infractionHeight < k.FirstConsumerHeight(ctx) {
+		k.standaloneStakingKeeper.Slash(ctx, addr, infractionHeight, power, slashFactor, infraction)
```

Comment:
nits: the `standaloneStakingKeeper`'s slash method doesn't use the infraction arg so it can be replaced by a the infraction type zero value.
```suggestion
		k.standaloneStakingKeeper.Slash(ctx, addr, infractionHeight, power, slashFactor, InfractionEmpty)
```

Reply:
nits: the `standaloneStakingKeeper`'s slash method doesn't use the infraction arg so it can be replaced by a the infraction type zero value.
```suggestion
		k.standaloneStakingKeeper.Slash(ctx, addr, infractionHeight, power, slashFactor, stakingtypes.InfractionEmpty)
```